docs/global/modules/core/pages/reference/component-bulk-model-sync-gradle.adoc

```diff
@@ -144,6 +144,10 @@ If the target branch does not exist on the model-server, it will be created.
 |Source model-server revision. Can be used instead of `repositoryId` and `branchName`.
 Only available in ServerSource.
 
+|`baseRevision`
+|String
+|Source model-server base revision. If specified, only the subtrees that changed between `baseRevision` and `revision` will be synchronized.
```

Comment:
Is this available / useful for both directions?

Reply:
For now only ServerSource. Added hint.